src/datasources/cache/__tests__/fake.cache.service.spec.ts

```diff
@@ -49,4 +50,52 @@ describe('FakeCacheService', () => {
 
     expect(target.keyCount()).toBe(0);
   });
+
+  it('deletes keys by pattern', async () => {
+    const prefix = faker.datatype.uuid();
+
+    // insert 5 items matching the pattern
+    await Promise.all(
+      range(5).map(async () =>
+        target.set(new CacheDir(`${prefix}${faker.datatype.uuid()}`, ''), ''),
+      ),
+    );
+
+    // insert 4 items not matching the pattern
+    await Promise.all(
+      range(4).map(async () =>
+        target.set(new CacheDir(`${faker.datatype.uuid()}`, ''), ''),
+      ),
+    );
+
+    await target.deleteByPattern(`${prefix}*`);
+
+    expect(target.keyCount()).toBe(4);
+  });
+
+  it('deletes keys by pattern (2)', async () => {
+    const prefix = faker.datatype.uuid();
+    const suffix = faker.datatype.uuid();
+
+    // insert 5 items matching the pattern
+    await Promise.all(
+      range(5).map(async () =>
+        target.set(
+          new CacheDir(`${prefix}_${faker.datatype.uuid()}_${suffix}`, ''),
+          '',
+        ),
+      ),
+    );
+
```

Comment:
nitpick: for the test setup we have been following the following "standard":

```
// setup \n
\n
// action \n
\n
// assertions
```

In this case I'd say it's fine to remove the newlines as it's part of the "setup"

Reply:
Changed in [54cf021](https://github.com/5afe/safe-client-gateway-nest/pull/288/commits/54cf021cea7b64d7e4246ae066d3c671209e3dde)